### ISSUE TITLE
client.info?

### ISSUE THREAD

**Issue by pwightman** [moomerman/twitter_oauth#42]
When using v0.4.4 of this gem, we have a variant of this code in the callback:

``` ruby
access_token = client.authorize(
  request_token["token"],
  request_token["secret"],
  :oauth_verifier => params[:oauth_verifier]
)

twitter_account = TwitterAccount.where( :twitter_user_id => client.info["id"] )
twitter_account.screen_name = client.info["screen_name"]
twitter_account.image_url = client.info["profile_image_url_https"]
```

But in upgrading to v0.4.91, `info` no longer appears to be a method on client. Where do we obtain info information now if not through that method?

**Comment by moomerman** [comment]
Sorry about that.  Twitter ended their v1 api today and I made some quick fixes to make it work.  I should have aliased the info method.  You can do the same thing by calling client.verify_credentials I will add the info alias back in though.

Richard

**Comment by moomerman** [comment]
info now included in v0.4.93 thanks to @filipebarcos
